2018/01/06 10:30:06 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1049] Unknown database 'navin_contacts'' in D:\xampp\htdocs\Contacts\framework\db\CDbConnection.php:399
Stack trace:
#0 D:\xampp\htdocs\Contacts\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 D:\xampp\htdocs\Contacts\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 D:\xampp\htdocs\Contacts\framework\base\CModule.php(394): CDbConnection->init()
#3 D:\xampp\htdocs\Contacts\framework\base\CApplication.php(459): CModule->getComponent('db')
#4 D:\xampp\htdocs\Contacts\framework\db\ar\CActiveRecord.php(639): CApplication->getDb()
#5 D:\xampp\htdocs\Contacts\framework\db\ar\CActiveRecord.php(2389): CActiveRecord->getDbConnection()
#6 D:\xampp\htdocs\Contacts\framework\db\ar\CActiveRecord.php(413): CActiveRecordMetaData->__construct(Object(Contacts))
#7 D:\xampp\htdocs\Contacts\framework\db\ar\CActiveRecord.php(79): CActiveRecord->getMetaData()
#8 D:\xampp\htdocs\Contacts\protected\controllers\ContactsController.php(9): CActiveRecord->__construct()
#9 D:\xampp\htdocs\Contacts\framework\web\actions\CInlineAction.php(49): ContactsController->actionIndex()
#10 D:\xampp\htdocs\Contacts\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 D:\xampp\htdocs\Contacts\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#12 D:\xampp\htdocs\Contacts\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 D:\xampp\htdocs\Contacts\framework\web\CWebApplication.php(282): CController->run('')
#14 D:\xampp\htdocs\Contacts\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#15 D:\xampp\htdocs\Contacts\framework\base\CApplication.php(185): CWebApplication->processRequest()
#16 D:\xampp\htdocs\Contacts\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/Contacts/
---
2018/01/06 10:31:28 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'status' in 'where clause'. The SQL statement executed was: SELECT * FROM `contacts` `t` WHERE status = 'active'.
in D:\xampp\htdocs\Contacts\protected\controllers\ContactsController.php (10)
in D:\xampp\htdocs\Contacts\index.php (13)
2018/01/06 10:31:28 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'status' in 'where clause'. The SQL statement executed was: SELECT * FROM `contacts` `t` WHERE status = 'active'' in D:\xampp\htdocs\Contacts\framework\db\CDbCommand.php:543
Stack trace:
#0 D:\xampp\htdocs\Contacts\framework\db\CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 D:\xampp\htdocs\Contacts\framework\db\ar\CActiveRecord.php(1359): CDbCommand->queryAll()
#2 D:\xampp\htdocs\Contacts\framework\db\ar\CActiveRecord.php(1478): CActiveRecord->query(Object(CDbCriteria), true)
#3 D:\xampp\htdocs\Contacts\protected\controllers\ContactsController.php(10): CActiveRecord->findAll(Array)
#4 D:\xampp\htdocs\Contacts\framework\web\actions\CInlineAction.php(49): ContactsController->actionIndex()
#5 D:\xampp\htdocs\Contacts\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 D:\xampp\htdocs\Contacts\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 D:\xampp\htdocs\Contacts\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 D:\xampp\htdocs\Contacts\framework\web\CWebApplication.php(282): CController->run('')
#9 D:\xampp\htdocs\Contacts\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#10 D:\xampp\htdocs\Contacts\framework\base\CApplication.php(185): CWebApplication->processRequest()
#11 D:\xampp\htdocs\Contacts\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/Contacts/
---
2018/01/06 10:32:26 [error] [exception.CException] exception 'CException' with message 'CAssetManager.basePath "D:\xampp\htdocs\Contacts\assets" is invalid. Please make sure the directory exists and is writable by the Web server process.' in D:\xampp\htdocs\Contacts\framework\web\CAssetManager.php:138
Stack trace:
#0 D:\xampp\htdocs\Contacts\framework\web\CAssetManager.php(123): CAssetManager->setBasePath('D:\\xampp\\htdocs...')
#1 D:\xampp\htdocs\Contacts\framework\web\CAssetManager.php(217): CAssetManager->getBasePath()
#2 D:\xampp\htdocs\Contacts\framework\zii\widgets\grid\CGridView.php(350): CAssetManager->publish('D:\\xampp\\htdocs...')
#3 D:\xampp\htdocs\Contacts\framework\web\CBaseController.php(147): CGridView->init()
#4 D:\xampp\htdocs\Contacts\framework\web\CBaseController.php(181): CBaseController->createWidget('zii.widgets.gri...', Array)
#5 D:\xampp\htdocs\Contacts\protected\views\contacts\index.php(78): CBaseController->widget('zii.widgets.gri...', Array)
#6 D:\xampp\htdocs\Contacts\framework\web\CBaseController.php(126): require('D:\\xampp\\htdocs...')
#7 D:\xampp\htdocs\Contacts\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\\xampp\\htdocs...', Array, true)
#8 D:\xampp\htdocs\Contacts\framework\web\CController.php(872): CBaseController->renderFile('D:\\xampp\\htdocs...', Array, true)
#9 D:\xampp\htdocs\Contacts\framework\web\CController.php(785): CController->renderPartial('index', Array, true)
#10 D:\xampp\htdocs\Contacts\protected\controllers\ContactsController.php(42): CController->render('index', Array)
#11 D:\xampp\htdocs\Contacts\framework\web\actions\CInlineAction.php(49): ContactsController->actionIndex()
#12 D:\xampp\htdocs\Contacts\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 D:\xampp\htdocs\Contacts\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#14 D:\xampp\htdocs\Contacts\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 D:\xampp\htdocs\Contacts\framework\web\CWebApplication.php(282): CController->run('')
#16 D:\xampp\htdocs\Contacts\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#17 D:\xampp\htdocs\Contacts\framework\base\CApplication.php(185): CWebApplication->processRequest()
#18 D:\xampp\htdocs\Contacts\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/Contacts/
---
2018/01/06 10:33:42 [warning] [application] Failed to set unsafe attribute "hobbies" of "Contacts".
in D:\xampp\htdocs\Contacts\protected\controllers\ContactsController.php (49)
in D:\xampp\htdocs\Contacts\index.php (13)
